src/wasm-delegations-fields.def

```diff
@@ -461,12 +461,14 @@ switch (DELEGATE_ID) {
   }
   case Expression::Id::MemorySizeId: {
     DELEGATE_START(MemorySize);
+    DELEGATE_FIELD_TYPE(MemorySize, ptrType);
     DELEGATE_END(MemorySize);
     break;
   }
   case Expression::Id::MemoryGrowId: {
     DELEGATE_START(MemoryGrow);
     DELEGATE_FIELD_CHILD(MemoryGrow, delta);
+    DELEGATE_FIELD_TYPE(MemoryGrow, ptrType);
```

Comment:
The order here should be flipped (as the stack traversal pushes first), that is, the new line should be before `delta`.